x-pack/plugins/security_solution/server/endpoint/utils/action_list_helpers.ts

```diff
@@ -70,15 +67,27 @@ export const getActions = async ({
     ...additionalFilters,
   ];
 
+  const must: SearchRequest = [
+    {
+      bool: {
+        filter: actionsFilters,
+      },
+    },
+  ];
+
+  if (userIds?.length) {
+    const kql = userIds.map((userId) => `user_id:*${userId}*`).join(' or ');
```

Comment:
Just so that we are all clear: this implies that the API does not allow for searching on exact names. It will always do a wildcard search. Is that the intent?

Would it be better if we allowed the user to enter values (user names) along with the wildcards?

Reply:
done 069ee756ea17a0024c7947707612e31516cbd620